discovery/chan_series.go

```diff
@@ -131,10 +132,24 @@ func (c *ChanSeries) UpdatesInHorizon(chain chainhash.Hash,
 
 		updates = append(updates, chanAnn)
 		if edge1 != nil {
-			updates = append(updates, edge1)
+			// We don't want to send channel updates that don't
+			// conform to the spec (anymore).
+			err := routing.ValidateChannelUpdateFields(0, edge1)
+			if err != nil {
+				log.Debugf("Invalid channel update %v: %v",
```

Comment:
Maybe this should be an error log?

Reply:
Changed it to an error log, I was not really sure here if we should surface it by default